src/lib/components/list/QItem.svelte

```diff
@@ -33,23 +34,25 @@
 
   $: separatorOptions = getContext<QListProps["separatorOptions"] | undefined>("separator");
 
-  $: isActionable = clickable === true || hasLink === true || tag === "label";
+  $: isActionable = clickable || hasLink || tag === "label";
 
-  $: isClickable = disable !== true && isActionable === true;
+  $: isClickable = isActionable && !disable;
 
   $: isActive = isRouteActive($Quaff.router, to);
 
-  $: classes = createClasses([
-    "q-item flex items-center",
-    $hasMultiplLines && "multiline",
-    dense && "dense",
-    hasLink && active && "q-item--active",
-    hasLink && active && activeClass,
-    isActive && "active",
-    isClickable && "wave",
-    linkClasses,
-    userClasses,
-  ]);
+  $: classes = createClasses(
+    [
+      $hasMultiplLines && "multiline",
```

Comment:
Can we use full words for this please? E.g. `hasMultipleLines`

Reply:
We sure can